blog-posts/github-billing-dashboard.md

```diff
@@ -0,0 +1,169 @@
+---
+path: "/blog/github-billing-dashboard/"
+date: "2021-07-15"
+title: "GitHub CSV Billing Dashboard"
+featuredImage: images/space.jpg
+author: "Fabian & Felix"
+authorSummary: "Developers at Satellytes"
+attribution:
+creator: upklyak
+source: https://www.freepik.com/free-vector/space-landscape-night_13748451.htm
+---
+
+
+
+> You can try out the dashboard here: [https://satellytes.github.io/github-billing-dashboard](https://satellytes.github.io/github-billing-dashboard)
+
+We use CI/CD pipelines and automatizations in every project we are responsible for. Especially Github Actions is becoming more popular, as it’s simple to use and growing in functionality. Over the last couple of months, we managed to create multiple projects and repositories that are using Github Actions extensively.
+
+Github Actions provides a generous free tier, so costs for smaller projects are often nonexistent. Once you reach the free tier limits, every Action (and other services) needs to be "paid by use" at the end of the month. Github aggregates all costs and shows them in your settings. But Github doesn't provide charts or a dashboard to get into the details of the costs, for example for historical data.
+
+This can be a problem if you have multiple projects, as it’s not obvious which service causes the costs. If you want to figure that out, you can get a CSV from Github, which is hard to read and analyze. We, therefore, decided to create a dashboard to visualize this CSV and show you the costs per repository and month.
+
+![screenshot of the billing settings in a Github account](./images/screenshot-of-gh-billing-settings.png)
+
+This is all the billing information you get from the Github Settings. No historical data, no expense per service.
+
+![screenshot of the Github billing dashboard with some charts](./images/screenshot-of-gh-dashboard.png)
+
+And this is what you get from the Github Billing Dashboard. Costs per service per month with historic data.
+
+This blog post gives you some insights into how we build it.
+
+## Technologies and Libraries
+
+The dashboard runs locally in the user's browser and thus does not send any data to external servers, even the most recently used files are stored only in the browser's local storage. The dashboard was mainly developed with React and Typescript. For the styling of the dashboard, we used styled-components. In addition, various libraries were used.
```

Comment:
>  In addition, various libraries were used.

Well either remove the sentence or clarify at least which kind of libraries if you don't want to refer to all of them. Especially because you are explaining details about `papaparse` but you don't mention it here? Neither is `Recharts` mentioned. 

I get the sentence now, it's like telling "we have used some libraries, and we will explain the interesting ones. I would list and summarize them while being here to prepare the user for the following sections.

Reply:
We have now written the libraries into a clear list. Hopefully, it is now clear to the reader what is meant by this sentence.

Reply:
Cool, that's how I imagined it. You could have linked the libraries here already too but the links in the details are enough. I'm probably only interested in the link when I'm reading the details anyway 👌